docs/index.md

```diff
@@ -1049,17 +1095,30 @@ The Google driver uses oAuth.  When creating the machine, you will have your bro
 
 Options:
 
- - `--google-zone`: The zone to launch the instance.  Default: `us-central1-a`
- - `--google-machine-type`: The type of instance.  Default: `f1-micro`
- - `--google-username`: The username to use for the instance.  Default: `docker-user`
- - `--google-project`: The name of your project to use when launching the instance.
+ - `--google-zone`: The zone to launch the instance.
+ - `--google-machine-type`: The type of instance.
+ - `--google-username`: The username to use for the instance.
+ - `--google-project`: **required** The name of your project to use when launching the instance.
  - `--google-auth-token`: Your oAuth token for the Google Cloud API.
```

Comment:
This flag args is auth token **file path**.
I think change the description, or change the flag name.